src/Avalonia.Controls/Converters/ProgressRingEndsToPathDataConverter.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Data.Converters;
+using Avalonia.Media;
+
+namespace Avalonia.Controls.Converters
+{
+    public class ProgressRingEndsToPathDataConverter : IMultiValueConverter
+    {
+        static readonly double TAU = 6.2831853071795862; //Math.Tau doesn't exist pre-.NET 5 :(
```

Comment:
const?

Reply:
...good catch, woops

Reply:
It seems to not be used anymore

Reply:
Yeah, we're not using that converter anymore...let me just get rid of it now